extensions/http-curl/sitetosite/HTTPProtocol.cpp

```diff
@@ -297,23 +305,9 @@ void HttpSiteToSiteClient::closeTransaction(const std::string &transactionID) {
 }
 
 void HttpSiteToSiteClient::deleteTransaction(std::string transactionID) {
-  std::shared_ptr<Transaction> transaction = NULL;
-
-  std::map<std::string, std::shared_ptr<Transaction> >::iterator it = this->known_transactions_.find(transactionID);
-
-  if (it == known_transactions_.end()) {
-    return;
-  } else {
-    transaction = it->second;
-  }
-
-  std::string append_str;
-  logger_->log_debug("Site2Site delete transaction %s", transaction->getUUIDStr());
-
```

Comment:
Good stuff. This should have been done when SiteToSiteClient was broken out into a base class but never was, so thanks for catching this.